code/modules/mob/living/carbon/human/life.dm

```diff
@@ -159,11 +151,6 @@
 		if(leg_clothes)
 			burning_items += leg_clothes
 
-		if(back)
-			burning_items += back
-		if(belt)
-			burning_items += belt
```

Comment:
backpack, belt, pocket stuff, id card, suit storage all somehow don't take damage when we're on fire. because balance...